docs/API.md

```diff
@@ -126,6 +126,17 @@ Basic data type: number
 
 A higher number means higher priority.
 
+This parameter is any integer that will be used by The Zonemaster Backend Worker daemon to sort the test requests from highest to lowest priority.
```

Comment:
The part about the Worker deamon sorting stuff is redundant in the presence of the previous sentence.